app/build.gradle

```diff
@@ -6,8 +6,8 @@ android {
         applicationId "com.finalsite.finalsiteacademy2"
         minSdkVersion 21
         targetSdkVersion 29
-        versionCode 4100400
-        versionName "4.10.4"
+        versionCode 4160200
+        versionName "4.16.2.6"
```

Comment:
Guess it won't matter since this is dynamically handled, but i think this means your settings.yml has `4.16.2` set as the version number so the build number won't increment properly. (we do a major minor and build number instead of major minor patch and build number)